src/main/steps/common/task-list/components/notification-banner/utils.ts

```diff
@@ -124,8 +124,9 @@ export const isPrimaryApplicant = (caseData: Partial<CaseWithId>, userDetails: U
 };
 
 export const isPartyServed = (caseData: Partial<CaseWithId>, userDetails: UserDetails): boolean => {
-  return (
+  return !!(
+    caseData.citizenApplicationPacks &&
     caseData.citizenApplicationPacks?.length !== 0 &&
-    getPartyDetails(caseData as CaseWithId, userDetails.id)?.id === caseData.citizenApplicationPacks?.[0].partyId
+    getPartyDetails(caseData as CaseWithId, userDetails.id)?.partyId === caseData.citizenApplicationPacks?.[0].partyId
```

Comment:
you dont have to check length !==0 & not have ? for [0] in applicationPacks

Reply:
Replaced check for caseData.citizenApplicationPacks with ? in caseData.citizenApplicationPacks?.length as there was a sonar issue